modules/structs/issue_tracked_time.go

```diff
@@ -8,23 +8,29 @@ import (
 	"time"
 )
 
+// AddTimeOption options for adding time to an issue
+type AddTimeOption struct {
+	// time in seconds
+	// required: true
+	Time int64 `json:"time" binding:"Required"`
+	// swagger:strfmt date-time
+	Created time.Time `json:"created"`
+	// User who spend the time (optional)
+	User string `json:"user_name"`
+}
+
 // TrackedTime worked time for an issue / pr
 type TrackedTime struct {
 	ID int64 `json:"id"`
 	// swagger:strfmt date-time
 	Created time.Time `json:"created"`
 	// Time in seconds
-	Time    int64 `json:"time"`
-	UserID  int64 `json:"user_id"`
-	IssueID int64 `json:"issue_id"`
+	Time     int64  `json:"time"`
+	UserID   int64  `json:"user_id"`
```

Comment:
can we mark `UserID` & `IssueID` as deprecated somehow?

Reply:
I wish - but could not found a option in swagger docu wich was efective ... so this can be removed if we make api_v2 or so ... :(

-> I'll leaf a comment so future maintainer will know :)